playbook/update_repo.yaml

```diff
@@ -38,6 +38,16 @@
         recurse: true
       register: files_normal
 
+    - name: Find GitHub Workflow files
+      find:
+        paths: "{{ template_dir }}/.github/workflows/"
+        excludes: "*.j2"
```

Comment:
Do we really want to exclude .j2 files here? The main workflow is a .j2 file and I can see .j2 being replaced in the renovate template as well...

Reply:
It should actually remain valid json, as we only replace within string literals.

Reply:
Good catch, fixed.

Reply:
We'll revisit this topic in a couple weeks when we did something to the template so it isn't valid json anymore and everything breaks :)